tonic/src/transport/channel/tls.rs

```diff
@@ -15,6 +15,7 @@ pub struct ClientTlsConfig {
     cert: Option<Certificate>,
     identity: Option<Identity>,
     rustls_raw: Option<tokio_rustls::rustls::ClientConfig>,
+    _certs_validation: CertsValidation,
```

Comment:
This should probably be a bool and should be feature gated.

```rust
#[cfg(feature = "tls-dangerous")]
cert_validation: bool
```

Reply:
It makes it really difficult to put that macro here considering I added a new parameter to `new_with_rustls_cert`.